test/test_ops.jl

```diff
@@ -7,7 +7,7 @@ end
 
 @testsection "tensor ops" begin
 for T in (Float32, Float64, F64), dim in (1,2,3)
-
+println("T = $T, dim = $dim")
```

Comment:
Ah, right. It was removed in #23 :) https://github.com/KristofferC/Tensors.jl/pull/23/files#diff-78a3de5eaf1c4bcf42b32436cba105ceL8